pkg/microservice/aslan/core/log/handler/log.go

```diff
@@ -128,3 +129,28 @@ func GetWorkflowBuildV3JobContainerLogs(c *gin.Context) {
 	// Use all lowercase job names to avoid subdomain errors
 	ctx.Resp, ctx.Err = logservice.GetWorkflowBuildV3JobContainerLogs(strings.ToLower(c.Param("workflowName")), c.Query("type"), taskID, ctx.Logger)
 }
+
+func GetScanningContainerLogs(c *gin.Context) {
+	ctx := internalhandler.NewContext(c)
+	defer func() { internalhandler.JSONResponse(c, ctx) }()
+
+	id := c.Param("id")
+	if id == "" {
+		ctx.Err = fmt.Errorf("id must be provided")
+		return
+	}
+
+	taskIDStr := c.Param("scan_id")
```

Comment:
Generally, camel case is used in query or param

Reply:
it is our own naming, not related to API or anything